internal/datacoord/compaction_test.go

```diff
@@ -346,7 +346,7 @@ func (s *CompactionPlanHandlerSuite) TestPickAnyNode() {
 	s.SetupTest()
 	nodeSlots := map[int64]int64{
 		100: 16,
-		101: 24,
+		101: 23,
```

Comment:
why 23 works

Reply:
the first pick
100: 16,
101: 24 -> 16 
the second pick may get 100 or 101

change to 23
23 - 8 -> 15 
the second pick must get 100